### PR TITLE
#63 Filter to issues added

### PR DESCRIPTION
**Pull request by ashwini571**
Change the name of MENTORS and PROJECTS accordingly in navbar.html

### CODE REVIEW

**Comment by ashwini571** [comment]
I'm unable to add the PR link on the Website .Kindly go through the problem

File: .idea/contri_original.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
```
Comment (dbads) [inline .idea/contri_original.iml]: Put .idea files into .gitignore

File: Projects/views.py [hunk 2]
```diff
@@ -12,16 +12,47 @@
...
+			try:
+				mentor = User.objects.get(username=val)
+				issues = Issues.objects.filter(mentor=mentor)
+			# else ordered point wise
+			except User.DoesNotExist:
+				issues = Issues.objects.all().order_by('points')
+
+		paginator = Paginator(issues, 15)  # Show 15 issues per page
+		page = request.GET.get('page', 1)
+		try:
```
Comment (dbads) [inline Projects/views.py]: Too much redendent code :smile: try not to have same line of codes twice

**Comment by ashwini571** [comment]
To be specific.. What needs to be minimized here? 😛

File: Projects/views.py (hunk 2 above)
Comment (dbads) [reply Projects/views.py]: Line no 31-39 and 46-54 have same lines of code

Comment (ashwini571) [reply Projects/views.py]: But both are for two different response.......the lines(46-54) are required when it is not POST request.
Without this how could possibly the home page will fetch issues. 

And everything it apart its working without any bug

Comment (dbads) [reply Projects/views.py]: Though GUI looks good but are you sure this is working ? 

It won't work :smile: 
Some bugs - 
1. For infinite scroll a get request is sent for fetching next issues, these new issues won't follow the filter properties as the request sent for fetching new issues on infinite scroll doesn't know what to filter? 
2. How will you filter all issues by "xyz" project of "abcd" mentor i.e one can filter only on the basis of one property.

Some suggestions - 
1. Try to have checkboxes, filter on the basis of all checked/unchecked properties
2. For infinite scroll to work you need to send all the checked attributes with page number, send the filter properties names in dictionary and use them while fetching new issues on infinite scroll
3. Use GET method, no need to use POST

You'll surely get partial points. For this PR to be merged fix the bugs :smiley:

**Pull request merged**